repo2docker/buildpacks/r.py

```diff
@@ -201,13 +201,14 @@ def get_build_scripts(self):
         We set the snapshot date used to install R libraries from based on the
         contents of runtime.txt.
         """
-        rstudio_url = "https://download2.rstudio.org/rstudio-server-1.1.419-amd64.deb"
+        # Via https://rstudio.com/products/rstudio/download-server/debian-ubuntu/
+        rstudio_url = "https://download2.rstudio.org/server/trusty/amd64/rstudio-server-1.2.5001-amd64.deb"
```

Comment:
```suggestion
        rstudio_url = "https://download2.rstudio.org/server/bionic/amd64/rstudio-server-1.2.5001-amd64.deb"
```

The repo2docker base image is based on Ubuntu "Bionic" so I think we should fetch this instead